docs/backbone.radio.md

```diff
@@ -85,7 +23,9 @@ access to this API.
   * [Listening to Requests on Objects](#listening-to-requests-on-objects)
   * [When to use Requests](#when-to-use-requests)
 
-## Channel
+## Radio Concepts
+
```

Comment:
I'd add a little text here that says:

```markdown
The `Radio` message bus exposes some core concepts:

* `Channel` - a namespace mechanism.
* `Request` - execute single functions in a different part of your application.
* `Event` - alert other parts of your application that something happened.
```

It'll give the reader an idea of the contents of the 'chapter' and the scope of what they're about to read.